core/base/ccTypes.cpp

```diff
@@ -388,16 +435,16 @@ void HSV::rgb(float fR, float fG, float fB, float fA)
         h = 360 + h;
     }
 
-    // Make hue value wrap around 360 (i.e. -390 degrees would be 30 wrapped)
-    h = abs(fmod(h, 360.0F));
-
     a = fA;
 }
 
-void HSV::color(float& fR, float& fG, float& fB) const
+void HSV::get(float& fR, float& fG, float& fB) const
 {
```

Comment:
You should avoid prefixing the parameter names with the type, as in "fR" etc.. There is no need for it, and it's actually bad practice to do so.

Just call them r, g, b, or red, green and blue.

Reply:
ok.